gulpfile.js

```diff
@@ -431,4 +436,23 @@ gulp.task('release', ['apps', 'clean-release'], function () {
     }
 });
 
+//Create installer files in ./installer
+gulp.task('installer', ['apps', 'clean-installer'], function () {
+    fs.mkdir(installerDir, '0775', function(err) {
+        if (err) {
+            if (err.code !== 'EEXIST') {
+                throw err;
+            }
+        }        
+    });
+
+    var platforms = getPlatforms();
+    console.log('Generating installer...');
+    
+    if (platforms.indexOf('win32') !== -1) {
+        child_process.execSync('makensis ./assets-windows/installer.nsi')        
```

Comment:
Have a look at https://www.npmjs.com/package/makensis. This will allow even neater integration.

Reply:
I didn't saw it. I will change it.